honeybee_grasshopper_radiance/src/HB Annual Glare Metrics.py

```diff
@@ -0,0 +1,136 @@
+# Honeybee: A Plugin for Environmental Analysis (GPL)
+# This file is part of Honeybee.
+#
+# Copyright (c) 2022, Ladybug Tools.
+# You should have received a copy of the GNU Affero General Public License
+# along with Honeybee; If not, see <http://www.gnu.org/licenses/>.
+# 
+# @license AGPL-3.0-or-later <https://spdx.org/licenses/AGPL-3.0-or-later>
+
+"""
+Calculate Annual Glare Metrics from result (.dgp) files.
+_
+- Glare Autonmy is a metric describing the percentage of occupied
+hours that each sensor is below the glare threshold.
+- Spatial Glare Autonomy is a metric describing the percentage of the sensor grid
+that is free glare according to the glare threshold and the target time. The sGA
+value is expressed as a percentage of area.
+
+-
+    Args:
+        _results: An list of annual Radiance result files from the "HB Imageless Annual
+            Glare" component (containing the .dgp files and the sun-up-hours.txt).
+            This can also be just the path to the folder containing these result
+            files.
+        _occ_sch_: An annual occupancy schedule as a Ladybug Data Collection or a HB-Energy
+            schedule object. This can also be the identifier of a schedule in
+            your HB-Energy schedule library. Any value in this schedule that is
+            0.1 or above will be considered occupied. If None, a schedule from
+            9AM to 5PM on weekdays will be used.
+        _threshold_: Threshhold for daylight autonomy (DA) in lux (default: 300).
+        _min_max_: A list for min, max illuminacne thresholds for useful daylight illuminance
+            in lux. (Default: (100, 3000)).
+        grid_filter_: The name of a grid or a pattern to filter the grids. For instance,
+            first_floor_* will simulate only the sensor grids that have an
+            identifier that starts with first_floor_. By default all the grids
+            will be processed.
+        mesh_: An optional list of Meshes that align with the meshes used in the imageless
+            annual glare recipe, which will be used to assign an area to each sensor.
+            If no mesh is connected here, it will be assumed that each sensor represents
+            an equal area to all of the others.
+        _target_time_: A minimum threshold of occupied time (eg. 95% of the time), above
+            which a given sensor passes and contributes to the spatial glare
+            autonomy. (Default: 95%).
```

Comment:
It looks like all of these docstrgings that determine the component descriptions were not updated. Please update them to match the new inputs.